Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
Probably unintended file

Reply:
Is there not a Pipfile in the top level of the repo? It was there when I inherited it but assumed it was from ours.